.gitignore

```diff
@@ -107,3 +107,6 @@ node_modules/
 
 # Ignore transpiled JS code
 django_fido/static/django_fido/js
+
```

Comment:
This should be in your user `.gitignore` and not in project one. This should remain language specific.

Reply:
Removed .idea from .gitignore